src/app/screens/Home/components/ArticleItem/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { View, Image } from 'react-native';
+import CustomText from '@components/CustomText';
+import { Article } from '@interfaces/articlesInterface';
+import { formatDate } from '@utils/dateUtils';
+
+import styles from './styles';
+
+interface Props {
+  item: Article;
+}
+
+function ArticleItem({
+  item: {
+    title,
+    description,
+    updatedAt,
+    author: { image, username }
+  }
+}: Props) {
+  return (
+    <View style={styles.container}>
+      <View style={styles.containerImage}>
+        <Image source={{ uri: image }} style={styles.image} />
```

Comment:
Andy, how are you?, You forgot to add the case when the url is empty

Reply:
maybe {image && ( <Image source={{ uri: image }} style={styles.image} />} or added  a custom image when no image in response, like this: <Image source={image?{ uri: image }:CUSTOM_IMAGE} style={styles.image} />